deps/icu-small/source/common/brkeng.cpp

```diff
@@ -78,7 +78,7 @@ UnhandledEngine::findBreaks( UText *text,
                              int32_t /* startPos */,
                              int32_t endPos,
                              UVector32 &/*foundBreaks*/ ) const {
-    UChar32 c = utext_current32(text);
+    UChar32 c = utext_current32(text); 
```

Comment:
Kind of odd that this ICU bump contains quite a lot of trailing whitespace changes.

Reply:
happens every.single.time.  I usually locally re-land with `--fix-whitespace=all` and then push the landed version back to the branch, but i messed something up in the process this time. 

upstream ICU still has trailing whitespace.